src/org/jetbrains/sbt/runner/SbtRunConfiguration.scala

```diff
@@ -78,6 +75,11 @@ class SbtRunConfiguration(val project: Project, val configurationFactory: Config
     envirnomentVariables.putAll(params.getEnvironmentVariables)
   }
 
+  def determineMainClass(launcherPath: String): String = {
+    val jf = new JarFile(new File(launcherPath))
+    val attributes = jf.getManifest.getMainAttributes
+    attributes.getValue("Main-Class")
```

Comment:
`getValue` returns null if no "Main-Class" attribute is present. Could you change the signature of `determineMainClass` so it will return `Option[String]` instead? And fix places below so that if `determineMainClass` returns `None` default main class (`xsbt.boot.Boot`) is used?